src/main/java/pro/cloudnode/smp/bankaccounts/POS.java

```diff
@@ -295,7 +296,10 @@ public static Optional<POS> get(final @NotNull String id) {
      */
     public static void openOwnerGui(final @NotNull Player player, final @NotNull Chest chest, final @NotNull POS pos) {
         final @NotNull ItemStack[] items = Arrays.stream(chest.getInventory().getStorageContents()).filter(Objects::nonNull).toArray(ItemStack[]::new);
-        final int extraRows = 1;
+        int extraRows = 1;
+        // check if the chest is totally full (#35)
+        if (items.length >= 54)
+            extraRows = -1;
```

Comment:
:/

extraRows is the rows we dedicated for buttons. we should prob Math.min the next line

Reply:
i'm not happy about this approach either, i'll try to find something better

Reply:
tried to tackle this in https://github.com/cloudnode-pro/BankAccounts/pull/41/commits/3ef2b143c8c979ebf98221dbaf83e688bb3fe95e , it works the same but preserves the extraRows